diagnostic.md

```diff
@@ -49,7 +54,12 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+// $base-color: #6495ed;
+
+@function tint($color, $percentage) {
+  @return mix($base-color, $color, $percentage);
+}
+
```

Comment:
It's just looking for the code to actually tint $cornflower:
`tint($cornflower, 20%);`